docs/developer/decision-records/2022-04-21-dpf-blob-data-transfer/README.md

```diff
@@ -0,0 +1,38 @@
+# Blob storage transfer
+
+ADR describing the blob storage transfer end to end flow between 2 participants.
+
+## Description
+
+The data-plane-azure-storage extension can be used on DPF to support blob transfers.
+A client can trigger a blob transfer on the consumer side via the Data Management API.
+The client needs to use the managedResources=true option in its HTTP request, otherwise the transfer will not work. Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.   
```

Comment:
nit: highlight the query string option
```suggestion
The client needs to use the `managedResources=true` option in its HTTP request, otherwise the transfer will not work. Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.   
```